segregation/util/compute_all.py

```diff
@@ -1,19 +1,20 @@
 """
-Profile Wrappers for Segregation measures
+Compute Several Segregation measures at once
 """
 
 __author__ = "Renan X. Cortes <renanc@ucr.edu> and Sergio J. Rey <sergio.rey@ucr.edu>"
 
+import pandas as pd
 from segregation.aspatial import *
 from segregation.spatial import *
 
 __all__ = [
-    'Profile_Aspatial_Segregation', 'Profile_Spatial_Segregation',
-    'Profile_Segregation'
+    'Compute_All_Aspatial_Segregation', 'Compute_All_Spatial_Segregation',
+    'Compute_All_Segregation'
 ]
 
 
-def _profile_aspatial_segregation(data, group_pop_var, total_pop_var):
+def _compute_all_aspatial_segregation(data, group_pop_var, total_pop_var):
```

Comment:
should we add an optional argument to specify a subset of indices to calculate (in case you dont need them *all*)?

Reply:
Yeah, I thought about that also, but I'm +1 to add a different function like `Compute_At` (I can open a new issue for this), and leave the current one as is. This would be similar to the guidelines of the summarize function of tidyverse (https://dplyr.tidyverse.org/reference/summarise_all.html). What do you think?

Reply:
works for me